src/lsp/lp_doc.ml

```diff
@@ -51,19 +51,19 @@ let option_default o1 d =
 let mk_error ~doc pos msg =
   LSP.mk_diagnostics ~uri:doc.uri ~version:doc.version [pos, 1, msg, None]
 
-let process_pstep (pstate, diags) tac =
+let process_pstep (pstate,expo,diags) tac =
```

Comment:
This diff suggests that expo should be part of the proof state, IIUC.

Reply:
Fixed in https://github.com/Deducteam/lambdapi/pull/422/commits/06a69ca0c614573effeb0eeaa5dbb0938f7ca56a